ingenialink/ethernet/network.py

```diff
@@ -39,20 +38,21 @@ class NetStatusListener(Thread):
     """
 
     def __init__(self, network: "EthernetNetwork", refresh_time: float = 0.25) -> None:
-        super(NetStatusListener, self).__init__()
+        super().__init__()
         self.__network = network
         self.__refresh_time = refresh_time
         self.__stop = False
         self.__max_unsuccessful_pings = MAX_NUM_UNSUCCESSFUL_PINGS
 
     def run(self) -> None:
+        """Check the network status."""
         while not self.__stop:
             for servo in self.__network.servos:
                 unsuccessful_pings = 0
                 servo_ip = servo.ip_address
                 servo_state = self.__network.get_servo_state(servo_ip)
                 while unsuccessful_pings < self.__max_unsuccessful_pings:
-                    response = servo.is_alive()  # TODO: Use ping after CAP-924 is fixed
+                    response = servo.is_alive()
```

Comment:
Why do you removed TODO comment, I known it is not the pretties way to remember an issue, but sometimes can help. What rule forbids it?

Reply:
[FIX002](https://docs.astral.sh/ruff/rules/line-contains-todo/#line-contains-todo-fix002).
I created issues and linked them to the code.

Reply:
Ok